experimental/packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -320,6 +323,22 @@ export class FetchInstrumentation extends InstrumentationBase<FetchInstrumentati
         }
         const spanData = plugin._prepareSpanData(url);
 
+        if (plugin.getConfig().measureRequestSize) {
+          web
+            .getFetchBodyLength(...args)
+            .then(length => {
+              if (!length) return;
+
+              createdSpan.setAttribute(
+                SEMATTRS_HTTP_REQUEST_CONTENT_LENGTH,
+                length
+              );
+            })
+            .catch(error => {
+              plugin._diag.error('getFetchBodyLength', error);
```

Comment:
nit: should this be an error or warn? -- to me it feels like a warning as it won't "stop" the telemetry from flowing and may just be an issue on a specific browser / environment.